README.md

```diff
@@ -4,13 +4,91 @@ The [Penumbra](https://penumbra.zone/) monorepo for all things web.
 
 ![ci status](https://github.com/penumbra-zone/web/actions/workflows/turbo-ci.yml/badge.svg?branch=main)
 
-### Technical Documentation
+This is a monolithic repository of Penumbra web code, a monorepo. Multiple apps,
+internal packages, and published packages are developed in this repository, to
+simplify work and make broad cross-package changes more feasible.
 
-[Visit docs](./docs/README.md) to see guiding principles, technical decisions, and general architecture of the apps and
-packages in this repo.
+To participate in the test network, use the browser extension
+[Prax](https://chrome.google.com/webstore/detail/penumbra-wallet/lkpmkhpnhknhmibgnmmhdhgdilepfghe)
+from the Chrome Web Store.
 
-### Links
+You can talk to us on [Discord](https://discord.gg/hKvkrqa3zC).
 
-- [Prax Wallet](https://chrome.google.com/webstore/detail/penumbra-wallet/lkpmkhpnhknhmibgnmmhdhgdilepfghe) - chrome
-  extension for custodying, proving, and signing transactions.
-- [Testnet Penumbra Dapp](https://app.testnet.penumbra.zone/) - Swap, stake, & send on Penumbra.
+## You might be looking for examples
+
+### [`@penumbra-zone/client` nextjs example](https://github.com/penumbra-zone/nextjs-penumbra-client-example)
+
+### [`@penumbra-zone/wasm` nextjs example](https://github.com/penumbra-zone/nextjs-penumbra-wasm-example)
+
+## What's in here
+
+### [Minifront](https://app.testnet.penumbra.zone/): Dapp to swap, stake, and send on the Penumbra testnet.
+
+### [Prax](https://chrome.google.com/webstore/detail/penumbra-wallet/lkpmkhpnhknhmibgnmmhdhgdilepfghe): Extension for Chrome that provides key custody, manages chain activity, and hosts services used by dapps.
+
+### [Status](https://grpc.testnet.penumbra.zone/): Public info dashboard for Penumbra nodes.
+
+### Packages
+
+**🌘
```

Comment:
Is this supposed to be a bullet list? 

It currently renders like this:
<img width="1038" alt="image" src="https://github.com/penumbra-zone/web/assets/1121544/2c197f5e-e216-4ae9-81cd-cb89f0e02e0b">

Reply:
i intentionally made it a single-line list to reduce vertical space